atomic_reactor/plugins/cachi2_postprocess.py

```diff
@@ -279,8 +279,10 @@ def inject_into_build_dir(
                 "copy method used for cachi2 build_dir_injecting: %s", copy_method.__name__)
 
             copytree(
-                remote_source.sources_path, dest_dir,
-                symlinks=True, copy_function=copy_method, dirs_exist_ok=True)
+                remote_source.sources_path, dest_dir, symlinks=True,
+                # Specifically for "remote-source.tar.gz"
+                ignore=ignore_patterns('*.tar.gz', '*.tgz'),
```

Comment:
doc says that's ignore function is called recursivelly in each dir, does it mean, it will also ignore files in deps or app?

Reply:
Yes, AFAICT

Reply:
Please amend commit message to reflect changes

Reply:
Already done